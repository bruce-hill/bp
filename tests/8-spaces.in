onetwo
one two
one    two
one	two
xxx
one;;;two
