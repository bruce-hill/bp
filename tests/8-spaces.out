onetwo
one two
one    two
one	two
